### PR TITLE
regularly update request triggers

### PR DESCRIPTION
**Pull request by Raycoms**
# Changes proposed in this pull request:
- Fixes the request triggers to update regularly even if not in blocked request state
-
-

Review please

### CODE REVIEW

File: src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java [hunk 1]
```diff
@@ -440,6 +439,28 @@ private IAIState updateVisualState()
         job.setNameTag(this.getState().toString());
         //Update torch, seeds etc. in chestbelt etc.
         updateRenderMetaData();
+
+        if ( getOwnBuilding().hasWorkerOpenRequests( worker.getCitizenData() ) )
```
Comment (Asherslab) [inline src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java:443]: The hell is this formatting you’re doing ray? Spaces absolutely everywhere

Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java:443]: spaces, where? I can't see them

Comment (Asherslab) [reply src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java:443]: Can’t tell if that’s sarcasm so here: 
E9FD0090-18CC-4E9B-9B00-4BAFD872F569

Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java:443]: Ahh those you mean =D Isn't that our default formatter?

Comment (Asherslab) [reply src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java:443]: No it certainly isn’t lol. That’s completely abnormal

**Pull request merged**